tests/modules/gke_hub/fixture/variables.tf

```diff
@@ -31,7 +31,7 @@ variable "features" {
     configmanagement             = true
     identityservice              = false
     multiclusteringress          = null
-    servicemesh                  = false
+    servicemesh                  = true
```

Comment:
Is there any reason to change the default for servicemesh to true?

Reply:
This is only for test fixture, so the test will cover also servicemesh.

Reply:
Ah sorry. I missed that this was the fixture.